src/main/resources/application.yml

```diff
@@ -126,14 +128,45 @@ nhais:
       +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
-    cypherToMailbox: ${NHAIS_MESH_CYPHER_TO_MAILBOX:XX11=nhais_mailbox
-      TES5=nhais_mailbox
-      YY21=nhais_mailbox
-      KC01=nhais_mailbox
-      WHI5=nhais_mailbox
-      SO01=nhais_mailbox
-      UNY5=nhais_mailbox}
-    scanMailboxIntervalInMilliseconds: ${NHAIS_SCAN_MAILBOX_INTERVAL_IN_MILLISECONDS:60000}
-    scanMailboxDelayInSeconds: ${NHAIS_SCAN_MAILBOX_DELAY_IN_SECONDS:300}
+    subCAcert: ${NHAIS_MESH_SUB_CA:-----BEGIN CERTIFICATE-----
```

Comment:
This is the OpenTest intermediate cert right? Since our defaults are for opentest should we omit this value?

Reply:
I've put fake-mesh ca.cert: https://github.com/jamespic/fake-mesh/blob/master/src/fake_mesh/ca.cert.pem

Reply:
I'll remove that